documentation/modules/ROOT/pages/02_basics.adoc

```diff
@@ -182,6 +182,6 @@ Press `r` and the tests will start running. You should see the status change dow
 All 1 test is passing (0 skipped), 1 test was run in 11705ms. Tests completed at 16:02:32.
 ----
 
-TIP: If you don’t want to have continuous testing enabled, you can change this by adding `quarkus.test.continuous-testing=disabled` in `application.properties`. 
+TIP: If you don’t want to have continuous testing enabled, you can change this by adding `quarkus.test.continuous-testing=disabled` to your `src/main/resources/application.properties`. 
```

Comment:
I don't think you presented the `applications.properties` yet so I think it's probably better to have the full path.